Show forking instructions in the example's README

warpech opened the pull request:
This PR is my attempt of implementing the point:

   3. Make these examples runnable by anyone who knows how to clone a git repo by writing actionable README.md files for every example.

from https://github.com/handsontable/handsontable/issues/8980.

Now looking at it, I am not sure if the forking information that I wrote here is that useful. @kirszenbaum, @AMBudnik WDYT? This will be displayed to anyone who hits the "Show source" button on the Demo page.

### Context
<!--- Why is this change required? What problem does it solve? -->

### How has this been tested?
<!--- Please describe in detail how you tested your changes (doesn't apply to translations). -->

### Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature or improvement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Additional language file or change to the existing one (translations)

### Related issue(s):
1. https://github.com/handsontable/handsontable/issues/8980
2.
3.

### Affected project(s):
- [ ] `handsontable`
- [ ] `@handsontable/angular`
- [ ] `@handsontable/react`
- [ ] `@handsontable/vue`
- [ ] `@handsontable/vue3`

### Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] I have reviewed the guidelines about [Contributing to Handsontable](https://github.com/handsontable/handsontable/blob/master/CONTRIBUTING.md) and I confirm that my code follows the code style of this project.
- [ ] I have signed the [Contributor License Agreement](https://docs.google.com/forms/d/e/1FAIpQLScpMq4swMelvw3-onxC8Jl29m0fVp5hpf7d1yQVklqVjGjWGA/viewform?c=0&w=1)
- [ ] My change requires a change to the documentation.

AMBudnik commented:
@warpech I like the idea of forking a single example. But I'm not sure if I understand the process. If you do not use

```
git clone https://github.com/handsontable/handsontable.git
```

then what should I do before I use the `cat README.md` in the first step?

warpech commented:
You are right that this is still needed. I have added this information to avoid confusion.

I also moved the forking instruction to the common `README.md` file to make the README's inside each example shorter.

warpech commented:
I copied the change to all files. I think it is good to go. If you would like to propose an improvement, please open an issue.

warpech commented:
CI shows that builds for some old versions fail for reasons not related to this PR. In my understanding, examples will not be deployed because of that. It doesn't matter really, because `README.md` files are intended for human reading, not deployment.
